index.html

```diff
@@ -24,14 +26,130 @@ <h1>I’m Mariane
      feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
     <a href="">LET’S CONNECT</a>
     <ul class="social_media">
-     <li><a href=""><img src="images/twiter.svg" alt=""></a></li>
-     <li><a href=""><img src="images/link.svg" alt=""></a></li>
-     <li><a href=""><img src="images/vector.svg" alt=""></a></li>
-     <li><a href=""><img src="images/vector1.svg" alt=""></a></li>
-     <li><a href=""><img src="images/vector2.svg" alt=""></a></li>
+     <li><a href=""><i class="fab fa-twitter"></i></a></li>
+     <li><a href=""><i class="fab fa-linkedin"></i></a></li>
+     <li><a href=""><i class="fab fa-medium-m"></i></a></li>
+     <li><a href=""><i class="fab fa-github"></i></a></li>
+     <li><a href=""><i class="fab fa-angellist"></i></a></li>
     </ul>
    </section>
+   <section class="snap_container">
+    <article>
```

Comment:
[OPTIONAL]: Please enure that the content of the project is aligned with `flexbox`